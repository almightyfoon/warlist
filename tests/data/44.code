c25b1b0y221G3k3We41G7_3C3z3z3B2_2W2m3jgu___-0d171S1U1U1q1q1q1q1q1q1r1L3q3C3z3z3B2_2_gu