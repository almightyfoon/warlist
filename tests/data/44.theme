{"allowed":[46,47,48,52,54,57,75,80,84,154,156,163,214,218,220,225,229,232,233,234,235,236,237,238,239,244,248,250,252,254,255,257,259,260,291,314,318,320,321,323,330,333,334,335,336,337,356,358,359,361,363,364,369,370,375,377,378,380,382,420,427,455,456,457,499,500,501,503,514,520,522,523,647,655,656,688,696,726,783,785,808,811,812,815,2048,2066,3060,3061,3062,3065,3077,3097,3098,3103,3104,3138,3153,3154,3158,3159,3172,3176,3177,3178,3180,3183,3188,3189,4007,4019,4037,4040,6015,6078,6140,6155,6160,6163,6172,6185,6186,6187,6188,6189,6190,6191,6192,6193,6302,6335,6356,6377,6382,6383,6392,6393,6403,6404,6405,6414,6420],"required":[330,358,369,3172,6356,6377,6383],"free":[346,358,359,360,361,362,367,369,372,373,375,784,3038,3172,4032,4033,6377,6382,6383]}