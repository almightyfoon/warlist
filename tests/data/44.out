https://conflictchamber.com/#c25b1b0y221G3k3We41G7_3C3z3z3B2_2W2mm2gu___-0d171S1U1U1q1q1q1q1q1q1r1Lme3C3z3z3B2_2_gu

Protectorate Army - 75 / 75 points

[Kreoss 1] High Exemplar Kreoss [+29]
 - Reckoner [16]
 - Redeemer [11]
Allegiant of the Order of the Fist [3]
Eiryss, Mage Hunter of Ios [7]
Initiate Tristan Durant [4]
 - Redeemer [11]
Taryn di la Rovissi, Llaelese Gun Mage [5]
The Covenant of Menoth [4]
Vassal of Menoth [3]
Vassal of Menoth [3]
Wrack [1]
Choir of Menoth (min) [4]
Deliverer Sunburst Crew [5]
Exemplar Errants (max) [16]
Visgoth Juviah Rhoven & Honor Guard [9]
Fuel Cache (Objective) [0]

Protectorate Army - 75 / 75 points
[Theme] The Creator's Might

[Amon 1] High Allegiant Amon Ad-Raza [+29]
 - Castigator [12]
 - Crusader [10]
 - Crusader [10]
 - Dervish [7]
 - Dervish [7]
 - Dervish [7]
 - Dervish [7]
 - Dervish [7]
 - Dervish [7]
 - Devout [9]
 - Vigilant [9]
 - Hierophant [0(3)]
The Covenant of Menoth [0(4)]
Vassal of Menoth [0(3)]
Vassal of Menoth [3]
Wrack [1]
Choir of Menoth (min) [4]
Choir of Menoth (min) [4]
Fuel Cache (Objective) [0]
